static/usage/breadcrumbs/styling/css-props/index.md

```diff
@@ -0,0 +1,30 @@
+import Playground from '@site/src/components/global/Playground';
+
+import javascript from './javascript.md';
+import vue from './vue.md';
+
+import reactTS from './react/react_ts.md';
+import reactCSS from './react/react_css.md';
+
+import angularHTML from './angular/angular_html.md';
+import angularCSS from './angular/angular_css.md';
+
+<Playground
+  code={{
+    javascript,
+    react: {
+      files: {
+        'src/main.tsx': reactTS,
+        'src/main.css': reactCSS
```

Comment:
~ Since the primary focus in these examples is the stylesheet (when there is multiple files), would it make sense to show the CSS file example first? The component playground example will render the files in the order they are defined in the object, so moving the CSS file above the tsx/html (for Angular) would render the stylesheet first.

Reply:
Good thinking!